pom.xml

```diff
@@ -120,6 +122,26 @@
                 <artifactId>farao-glsk-document-api</artifactId>
                 <version>${farao.core.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.farao-community.farao</groupId>
+                <artifactId>farao-crac-creator-api</artifactId>
+                <version>${farao.core.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.rte-france.farao</groupId>
+                <artifactId>farao-data-fb-constraint</artifactId>
+                <version>${farao.rte.core.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.farao-community.farao</groupId>
+                <artifactId>gridcapa-rao-runner-spring-boot-starter</artifactId>
```

Comment:
I don't think you need this one as it is already defined in gridcapa-core-valid-app pom file

Reply:
No we do need it. We can discuss it